unlock-app/src/components/interface/checkout/main/Minting.tsx

```diff
@@ -195,10 +191,9 @@ export function Minting({
                     network={lock!.network}
                     lockAddress={lock!.address}
                     tokenId={tokenId}
-                    image={tokenImage}
                     name={lock!.name}
                     handlePassUrl={(url: string) => {
-                      window.open(url, '_')
+                      window.location.assign(url)
```

Comment:
Does that open in a new tab though? Maybe ok...

Reply:
Yeah. Alternative workaround would be to pop up a window when user clicks immediately and then update it later but that is also confusing UX.